File: fix3.py
with open('src/app/(dashboard)/admin/pesquisas/page.tsx', 'r', encoding='utf-8') as f:
    content = f.read()

content = content.replace(
    "{p.opcoes.slice(0, 3).map((o, i) => (",
    "{(p.perguntas?.[0]?.opcoes || p.opcoes || []).slice(0, 3).map((o, i) => ("
)

content = content.replace(
    "{p.opcoes.length > 3 && <div className=\"text-xs text-gray-400 pl-3.5 pt-1\">+{p.opcoes.length - 3} opções...</div>}",
    "{(p.perguntas?.[0]?.opcoes || p.opcoes || []).length > 3 && <div className=\"text-xs text-gray-400 pl-3.5 pt-1\">+{(p.perguntas?.[0]?.opcoes || p.opcoes || []).length - 3} opções...</div>}"
)

content = content.replace(
    "{p.opcoes.reduce((s, o) => s + o.votos, 0)}",
    "{p.respostas?.length || 0}"
)

with open('src/app/(dashboard)/admin/pesquisas/page.tsx', 'w', encoding='utf-8') as f:
    f.write(content)

File: scratch/fix3.js
const fs = require('fs');
const file = 'src/app/(dashboard)/admin/pesquisas/page.tsx';
let content = fs.readFileSync(file, 'utf8');

const bad = `if (e.target.closest('button')) return;
    setEditandoId(p.id);
    setNome(p.nome);
    setTipo(p.tipo);
    setStatus(p.status);
    setInduzida(p.induzida);
    setMultiSelect(p.multiSelect || false);
    setExigeMorador(p.exigeMorador || false);
    setIdadeMinima(p.idadeMinima?.toString() || '');
    setMetaDiaria(p.metaDiaria || 50);
    setMetaEntrevistas(p.metaEntrevistas || 1000);
    setOpcoes([...p.opcoes]);`;

const good = `if ((e.target as HTMLElement).closest('button')) return;
    setEditandoId(p.id);
    setNovaPesquisa({
      nome: p.nome,
      tipo: p.tipo,
      induzida: p.induzida,
      multiSelect: p.multiSelect || false,
      metaDiaria: p.metaDiaria || 50,
      metaEntrevistas: p.metaEntrevistas || 1000,
      exigeMorador: p.exigeMorador || false,
      idadeMinima: p.idadeMinima || 16
    });
    setNovasOpcoes(p.opcoes.map(o => ({ nome: o.nome, partido: o.partido || '', foto: o.foto || '' })));
    setEntrevSelecionados([...p.entrevistadores]);`;

content = content.replace(bad, good);

fs.writeFileSync(file, content, 'utf8');
